release/test_bom.sh

```diff
@@ -40,14 +48,29 @@ function set_up {
 	export _RELEASE_ROOT_DIR="${PWD}"
 
 	export _ARTIFACT_RC_VERSION="${_PRODUCT_VERSION}-${_BUILD_TIMESTAMP}"
-	export _PROJECTS_DIR="${_RELEASE_ROOT_DIR}"/../..
 	export _RELEASE_TOOL_DIR="${_RELEASE_ROOT_DIR}"
 
-	if [ ! -d "${_PROJECTS_DIR}/liferay-portal-ee" ]
+	if [ -n "${LIFERAY_RELEASE_GITHUB_PAT}" ]
 	then
-		echo -e "The directory ${_PROJECTS_DIR}/liferay-portal-ee does not exist.\n"
+		export _PROJECTS_DIR="${PWD}"
+
+		local sha=$(_get_tag_sha "liferay-release" "liferay-portal-ee" "${_PRODUCT_VERSION}" "${LIFERAY_RELEASE_GITHUB_PAT}") 
+
+		_get_github_repository_zip "liferay-release" "liferay-portal-ee" "${_PRODUCT_VERSION}" "${LIFERAY_RELEASE_GITHUB_PAT}"
+
+		unzip -q "repository.zip"
+
+		mv "liferay-release-liferay-portal-ee-${sha}" "liferay-portal-ee"
```

Comment:
@lucasmiranda0, can you create a dedicated function to hold these lines of code since you're using these same lines in 2 different places?